lib/index.tsx

```diff
@@ -27,11 +38,33 @@ class Stripe {
     this._stripeInitialized = true;
   }
 
-  confirmPayment(clientSecret: string, cardDetails: CardDetails): Promise<PaymentResult> {
-    return StripePayments.confirmPayment(clientSecret, cardDetails)
+  confirmPaymentWithCardParams(clientSecret: string, cardParams: CardParams): Promise<PaymentResult> {
+    return StripePayments.confirmPaymentWithCardParams(clientSecret, cardParams)
+  }
+
+  confirmPaymentWithPaymentMethodId(clientSecret: string, paymentMethodId: string): Promise<PaymentResult> {
+    return StripePayments.confirmPaymentWithPaymentMethodId(clientSecret, paymentMethodId);
+  }
+
+  async confirmSetup(clientSecret: string, cardParams: CardParams): Promise<SetupIntentResult>{
+    const nativeSetupIntentResult = await StripePayments.confirmSetup(clientSecret, cardParams);
+    const cardNumber = cardParams.number;
+    const cardType = creditCardType(cardNumber);
```

Comment:
I'm wondering if retrieval of card type (brand) should be part of this library. There are a few reasons:
- Stripe already has method to get card type, of course it it's a separate method, but that looks like a valid separation of concerns
- I'd like to avoid dependencies (except obviously stripe)

Thinking further about use cases - after confirmation list of cards will still be fetched from back-end (to get full list of cards) therefore it's not necessary to have this IMHO

Reply:
@viktorasl we could move the card type retrieval to use the native stripe method as part of this method or a separate method.

Reply:
separate method would be more appropriate I think